File: leetcode-submissions/2016_Maximum_Difference_Between_Increasing_Elements.cpp
class Solution {
public:
    int maximumDifference(vector<int>& nums) {
        int maxDiff = INT_MIN ;
        int n = nums.size();
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]<nums[j]){
                    maxDiff = max(maxDiff , nums[j]- nums[i]);
                }
            }
        }
        return maxDiff==INT_MIN?-1:maxDiff;
    }
};

File: leetcode-submissions/2016_Maximum_Difference_Between_Increasing_Elements.ts
function maximumDifference(nums: number[]): number {
    let maxDiff : number = Number.MIN_SAFE_INTEGER;
    const n : number = nums.length ;

    for(let i:number =0 ;i<n;i++ ){
        for(let j:number =i+1;j<n;j++){
            maxDiff = Math.max(maxDiff , nums[j]-nums[i]);
        }
    }
    return maxDiff ;
};